e2e/portalicious/primeng-components/PrimeDropdown.ts

```diff
@@ -15,4 +15,8 @@ export class PrimeDropdown {
     await this.dropdownOpener.click();
     await this.dropdownWrapper.getByRole('option', { name: label }).click();
   }
+
+  async selectMenuItem({ label }: { label: string }) {
+    await this.page.getByRole('menuitem', { name: label }).click();
+  }
```

Comment:
What you're referring to as as dropdown isn't actually a Dropdown, so this code shouldn't be here.

```suggestion
```

See my comment in BasePage for more info